src/index.js

```diff
@@ -1,2 +1,40 @@
-// Import styles
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225524171-42abf84c-18ef-4955-b1ae-05f6cee66756.png)

- [ ] Since you don't have anything for this section so it would be great if you remove this Deployment sub-section to make your readme file clean and professional 😊